src/app/body/task-details/activity/activity.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="card mt-2" *ngIf="(userObservable | async) || (activity.Uid == 'defaultUser')">
+<div class="card mt-3 mx-2" *ngIf="(userObservable | async) || (activity.Uid == 'defaultUser')">
     <div class="card-body row">
```

Comment:
Instead of margin you can use padding

Reply:
Padding can't be used here, we are making spaces between outer box and each activity action box, so won't work.

Reply:
Let's make it without margin